jykuo-love-shiritori/twp
# [backend] check if user is enabled when login

Fix #116 

### Changes


As listed in title

### Testing Approach (if unconventional)


- Disable a user
- Try login
- Login should fail

### Additional context


N/A
